docs/source/compatibility.rst

```diff
@@ -68,6 +68,37 @@ The actual change should be done in the following steps:
 - At the major update announced in the above update, change the API.
 
 
+Experimental APIs
+-----------------
+
+Thanks to many contributors, we have introduced many new features to Chainer.
+But sometimes it was only after we released the new features that we noticed that their APIs are not appropriate.
+The objective of experimental APIs is to alleviate the design failure of them.
+
+Any newly added API can be marked as *experimental*.
+Any API that is not experimental is called *stable* in this document.
+
+.. note::
+    Undocumented behaviors are not considered as APIs. So they are not experimental nor stable.
+    The treatment of undocumented behaviors are descibed in :ref:`undocumented_behavior` section.
+
+Chainer can change the interfaces and documents of experimental APIs at **any** version up.
+This change is not considered as a break of backward compatibility.
+Chainer can remove experimental mark of APIs and make them stable at any **minor** or **major** version up.
```

Comment:
Wording: I suggest to replace

Chainer can remove experimental mark of APIs and make them stable at any **minor** or **major** version up.

with the following:

Chainer can promote an experimental API to become stable at any **minor** or **major** version up.